content/docs/command-reference/remote/modify.md

```diff
@@ -108,6 +108,20 @@ these settings, you could use the following options:
   $ dvc remote modify myremote endpointurl https://myendpoint.com
   ```
 
+- `access_key_id` - (optional) AWS Access Key ID. Overrides credentials from
+  `credentialpath`:
+
+  ```dvc
+  $ dvc remote modify myremote access_key_id my-access-key-id
+  ```
+
+- `secret_access_key` - (optional) AWS Secret Access Key. Overrides credentials
+  from `credentialpath`:
```

Comment:
But actually... All of these settings are optional, even `credentialpath`. So why/when does `credentialpath` need to be overridden?

From the section's description: "To override some of these settings, you could use the following options".

Reply:
Is it just an alternative to `credentialpath`?

Reply:
https://github.com/iterative/dvc/issues/4175

Reply:
Yes, all of these are optional except for the url. You can override default credentialpath (~/.aws/credentials) when you want to use a different creds file.

Reply:
Thanks, I get it. My questions are geared towards improving the explanation for the users.

So maybe `credentialpath` should state it is the default AWS credentials file if omitted, or something like this? Another option is to say these new options can be used "instead of `credentialpath`" (and not call them optional). The way they're described now may be confusing.

Reply:
````suggestion
- `access_key_id` - AWS Access Key ID. May be used along with
  `secret_access_key` instead of `credentialpath`:

  ```dvc
  $ dvc remote modify myremote access_key_id my-access-key-id
  ```

- `secret_access_key` - AWS Secret Access Key. May be used along with
  `access_key_id` instead of `credentialpath`:
````

Reply:
FTFY I think ^ just not sure it'll pass the formatting check.

Reply:
These config options will override not only what is in credentialpath but also what is in the env or boto3-specific config, so the hierarchy is more complex and, ideally, it should be described (a note about us using boto3 and a reference to https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html might suffice) once for the whole s3-related section. We could create a ticket for that for later, but for now just say that these newly added options are optional.

Reply:
> We could create a ticket for that for later, but for now just say that these newly added options are optional.

Sure, feel free to open the ticket. Seems like a good note to add. For now I merged this. It does already imply all these settings are optional at the beginning of the section, and per the nature of `dvc remote modify`.